src/main/java/com/bp3x/raidbot/commands/lfg/RemindCommand.java

```diff
@@ -27,35 +29,51 @@ protected void execute(CommandEvent event) {
         log.info("Reminding for " + event.getArgs());
 
         String[] args = event.getArgs().split("\\s+");
-        if (args.length == 2) {
+        if (args.length == 2 && (args[1].equalsIgnoreCase("accepted") || args[1].equalsIgnoreCase("tentative"))) {
```

Comment:
Maybe the accepted and tentative check should be done in a nested if statement? This way, the error messages can be a bit more descriptive. 
```
if (args.length == 2) {
    if (args[1] .equals("accepted") || args[1].equals("tentative")) {
        // do things
    } else {
        print("use accepted or tentative");
    }

} else { print("You're missing some args")```